packages/accounts/src/msca/plugins/multi-owner.ts

```diff
@@ -1,6 +1,11 @@
 import { type GetFunctionArgs, encodeFunctionData } from "viem";
 import type { Plugin } from "./types";
-import type { BaseSmartContractAccount } from "@alchemy/aa-core";
+import type { MSCA } from "../builder";
+import type {
+  ISmartContractAccount,
+  ISmartAccountProvider,
+  SupportedTransports,
```

Comment:
nit q: should this just be imported from viem?

Reply:
This is something we define specifically on our end because it's a smaller subset of the `viem` transports type